docs/reference/cwk_path_join.md

```diff
@@ -3,7 +3,7 @@ title: cwk_path_join
 description: Joins two paths together.
```

Comment:
not sure why this is showing as modified?

Reply:
@xansec  I'm not sure... Resolved